devito/operator/profiling.py

```diff
@@ -367,19 +364,8 @@ class AdvisorProfiler(AdvancedProfiler):
     _ext_calls = [_api_resume, _api_pause]
 
     def __init__(self, name):
-        self.path = locate_intel_advisor()
-        if self.path is None:
-            self.initialized = False
-        else:
-            super().__init__(name)
```

Comment:
I prefer to keep this logic here rather than in compiler.py, which shouldn't be aware of profiling